doc/contribute-to-core-lightning/release-checklist.md

```diff
@@ -46,7 +46,7 @@ Here's a checklist for the release process.
 ## Releasing -rc2, ..., -rcN
 
 1. Update CHANGELOG.md by changing rc(N-1) to rcN. Update the changelog list with information from newly merged PRs also.
-2. Update the package versions: `make update-versions NEW_VERSION=<VERSION>`
+2. Update the package versions: `make update-versions NEW_VERSION=<VERSION>-rcN`
```

Comment:
```suggestion
2. Update the package versions: `make update-versions NEW_VERSION=<VERSION>rcN`
```
Keep this consistent with the first one.  Pyln-testing now looks for the hyphen as a separator for the commit hash, so probably better to leave it off.